libs/net/dal/Services/ReportService.cs

```diff
@@ -797,7 +797,8 @@ public IEnumerable<ReportInstance> GetLatestInstances(int id, int? ownerId = nul
             : Array.Empty<long>();
 
         // When an auto report runs it may need to exclude content in the currently unsent report.
-        if (currentInstance != null && currentInstance.SentOn.HasValue == false && reportSettings.Content.ExcludeContentInUnsentReport)
+        if (currentInstance != null && currentInstance.SentOn.HasValue == false
+            && (reportSettings.Content.ExcludeContentInUnsentReport || !reportSettings.Content.ClearOnStartNewReport))
```

Comment:
Add the current instance content to the exclude list, so that subsequent requests for content knows to look for new content.